server/chat-commands.js

```diff
@@ -4105,6 +4105,13 @@ const commands = {
 		}));
 	},
 
+	hidereplay: function (target, room, user, connection) {
```

Comment:
This comes from tours private code, at the request of the TDs. Its strictly less powerful than `/hideroom`, so doesnt seem to harmful to support for everyone, though it would likely only be of use to forced public users